src/Test/Utilities/Portable/Diagnostics/DiagnosticDescription.cs

```diff
@@ -106,6 +110,7 @@ private IEnumerable<string> GetArgumentsAsStrings()
             _errorCodeType = errorCodeType ?? code.GetType();
             _defaultSeverityOpt = defaultSeverityOpt;
             _effectiveSeverityOpt = effectiveSeverityOpt;
+            _isSuppressed = isSuppressed;
         }
 
         public DiagnosticDescription(Diagnostic d, bool errorCodeOnly, bool includeDefaultSeverity = false, bool includeEffectiveSeverity = false)
```

Comment:
We should add `bool isSuppressed = false` even to this constructor for consistency.

Reply:
Fixed.